apps/console/src/index.d.ts

```diff
@@ -0,0 +1,48 @@
+export {};
+
+declare global {
```

Comment:
Let's use https://www.npmjs.com/package/@analytics/segment instead of directly using the Segment stcript